src/v3/src/transformer/field/attributes.ts

```diff
@@ -30,37 +31,66 @@ const autocompleteValueMap = new Map<string, AutoCompleteValue>([
   ['email', 'email'],
 ]);
 
+const inputModeValueMap = new Map<string, InputModeValue>([
+  ['passcode', 'numeric'],
+  ['totp', 'numeric'],
+  ['email', 'email'],
+  ['phoneNumber', 'tel'],
+]);
+
 const getKeyFromMap = (
-  map: Map<string, AutoCompleteValue>,
+  map: Map<string, AutoCompleteValue> | Map<string, InputModeValue>,
```

Comment:
It doesn't seem like the generic type of `Map` matters, so could this be `Map<string, unknown>`?

Reply:
Great catch, yea that function doesn't use the value at all so can totally do that. Will update, thx!